spec/griddler/email_spec.rb

```diff
@@ -615,6 +615,19 @@ def header_from_email(header)
     expect(email.to).to eq [expected]
     expect(email.from).to eq expected
   end
+
+  it 'handles invalid UTF-8 characters' do
+    email = Griddler::Email.new(
+      text: 'hi', 
+      to: ["\xc0\xc1\xf5\xfa\xfe\xff #{@full_address}"],
+      from: "\xc0\xc1\xf5\xfa\xfe\xff #{@full_address}")
+    expected = @address_components.merge(
+      full: "ÀÁõúþÿ Bob <bob@example.com>", 
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Trailing whitespace detected.